eos-openqa-client/README.md

```diff
@@ -0,0 +1,7 @@
+The following two files need to be manually synced from eos-image-builder, as
+we can’t do a narrow clone with a submodule to only get the lib/ subdirectory
+from eos-image-builder. Also, eos-image-builder is private but eos-openqa-tests
+is public.
```

Comment:
I was going to suggest using `git subtree` because I thought it supported merging a subdir of another repo to a subdir of this one; but it doesn't. ☹